bundles/org.openhab.persistence.dynamodb/src/main/feature/feature.xml

```diff
@@ -5,6 +5,9 @@
 	<feature name="openhab-persistence-dynamodb" description="DynamoDB Persistence" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
 		<feature dependency="true">openhab.tp-jackson</feature>
+		<feature dependency="true">openhab.tp-netty</feature>
+		<!-- netty-codec-http2 version has been aligned with the openhab.tp-netty feature -->
+		<bundle dependency="true">mvn:io.netty/netty-codec-http2/4.1.42.Final</bundle>
```

Comment:
Can we use the ${netty.version} here?

Reply:
No because the property is defined only in the dynamodb project whereas the features are assembled and build in the features project. I've created https://github.com/openhab/openhab-core/pull/2257 so you can remove netty-codec-http2 from the add-on feature when it is merged.

Reply:
Done https://github.com/openhab/openhab-addons/pull/9937/commits/f961721ff737cbf4dc9c3833d00d5ed2b8e2fded

Thanks!